tests/console/system_prepare.pm

```diff
@@ -23,6 +24,18 @@ sub run {
 
     ensure_serialdev_permissions;
 
+    # Configure serial consoles for virtio support
+    # poo#18860 Enable console on hvc0 on SLES < 12-SP2
+    # poo#44699 Enable console on hvc1 to fix login issues on ppc64le
+    if (!check_var('VIRTIO_CONSOLE', 0)) {
```

Comment:
```suggestion
    if (check_var('VIRTIO_CONSOLE', 1)) {
```

or even 
```suggestion
    if (get_var('VIRTIO_CONSOLE')) {
```

Reply:
Thank you. I hope it will work exactly the same way.